src/rfkill/rfkill.c

```diff
@@ -280,10 +280,6 @@ static int run(int argc, char *argv[]) {
 
         umask(0022);
 
-        r = mkdir_p("/var/lib/systemd/rfkill", 0755);
-        if (r < 0)
-                return log_error_errno(r, "Failed to create rfkill directory: %m");
-
```

Comment:
The only doubt I have is whether this directory should exists even if we don't write anything there. But it doesn't seem so, i.e. nobody should care about this directory but us.